src/Actions/actions.js

```diff
@@ -120,10 +142,37 @@ function fetchCurrentPostData(postID) {
     }
 }
 
+function validateUsernameBackend(username) {
+    return {
+        type: 'VALIDATE_USERNAME_BACKEND',
+        data: {username}
```

Comment:
inconsistent how this one is wrapped in an object but the others are just `data`

Reply:
I had a hard time getting the username in the backend when it was not wrapped (BACKEND: <no wrap> --> username = request.json ;;;  <wrapped> --> username = request.json['username'])  any idea I can get the data in the backend without the wrap?

Reply:
i'm not sure what u mean. just leave it for now